packages/wasm/package.json

```diff
@@ -32,13 +32,13 @@
     "build": "run-p build:cjs build:esm build:bundle build:types",
     "build:bundle": "rollup --config",
     "build:cjs": "tsc -p tsconfig.cjs.json",
-    "build:dev": "run-s build:cjs build:esm build:types",
+    "build:dev": "run-p build:cjs build:esm build:types",
     "build:es5": "yarn build:cjs # *** backwards compatibility - remove in v7 ***",
     "build:esm": "tsc -p tsconfig.esm.json",
     "build:types": "tsc -p tsconfig.types.json",
-    "build:watch": "run-p build:cjs:watch build:esm:watch build:types:watch",
+    "build:watch": "run-p build:cjs:watch build:esm:watch build:bundle:watch build:types:watch",
```

Comment:
Heads up: `build:bundle:watch` doesn't exist in this file

Reply:
Heh. Shows you how often I run the full `build:watch`. Thanks for catching that.